src/models/searchservice/response/AddressDirectAnswer.ts

```diff
@@ -0,0 +1,28 @@
+import { BuiltInFieldType } from './directanswer/BuiltInFieldType';
+import { DirectAnswer } from './directanswer/DirectAnswer';
+
+/**
+ * A direct answer {@link DirectAnswer} interface with address value.
```

Comment:
can this be reworded? right now, the text would read "A direct answer DirectAnswer interface with ..."